app/src/main/res/values/strings.xml

```diff
@@ -47,6 +47,7 @@
     <string name="settings_excluded_apps">Excluded apps</string>
     <string name="ui_excluded_apps">View or edit apps excluded from KISS</string>
     <string name="ui_excluded_from_history_apps">View or edit apps excluded from history</string>
+    <string name="ui_excluded_from_shortcuts_apps">View or edit apps excluded from shortcuts</string>
```

Comment:
sounds somewhat misleading IMO, because not any app will be excluded from shortcuts, but shortcuts for specific apps are excluded from KISS if i read code right.

Reply:
That's correct - but I don't want to make the text too long, because the text doesn't wrap (it just crops if the screen isn't wide enough). Do you have any suggestions for the text? I originally wanted to do something like: "View or edit apps excluded from showing shortcuts"

Reply:
I've just gone for "View or edit apps excluded from showing shortcuts" for now. Maybe we can fix the line breaking in the future

Reply:
i see, thx for clarification.
it's hard to find something short here.